### PR TITLE
Fix old wavesblockexplorer.

### PR DESCRIPTION
**Pull request by username1565**
Add files to successfully build and run this old wavesblockexplorer.
See this issue: https://github.com/wavesplatform/WavesExplorerLite/issues/29

Also, I want some waves here: 3PHwTZasBNSnDJNP68v7Awcs3paRLTHnoao

### CODE REVIEW

File: BUILD_yarn-install.bat [hunk 1]
```diff
@@ -0,0 +1,3 @@
+cd %~dp0
```
Comment (beregovoy68) [inline BUILD_yarn-install.bat]: A script for automating `yarn install`?

Comment (username1565) [reply BUILD_yarn-install.bat]: Yes, it is. Just run by double-click.
`yarn` was been pre-installed, before run this .bat`-file on Windows 8.1 Pro.

File: src/js/bundle-1.1.8.js [hunk 2]
```diff
@@ -0,0 +1,1895 @@
+(function () {
```
[the remark this answers is not in the document]
Comment (username1565) [reply src/js/bundle-1.1.8.js]: This files not autogenerated when I try to compile old wavesblockexplorer on windows 8.1.
Also, this not contains inside `index.html`.
So I just did copy this from https://wavesblockexplorer.com/ but now there is new wavesexplorer installed, after their server crush.
I did add this files to this repositary to do easy compile and run this all by double-click on `.bat`-files. See source code.

File: src/js/templates-1.1.8.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
... <div class="panel-body" ng-show="ctrl.dataArray.length == 0">{{ctrl.dataMessage}}</div>\n            </uib-tab>\n            <uib-tab index="4" heading="Script" select="ctrl.loadScript()">\n                <table ng-show="ctrl.scripts.length > 0" class="table blockDetails">\n                    <thead>\n                    <tr class="active">\n                        <th>Field</th>\n                        <th>Value</th>\n                    </tr>\n                    </thead>\n                    <tbody ng-repeat="item in ctrl.scripts">\n                        <tr>\n                            <td>Compiled Script</td>\n                            <td><waves-compiled-script text="item.script"></waves-compiled-script></td>\n                        </tr>\n                        <tr>\n                            <td>Script Text</td>\n                            <td><waves-script text="item.scriptText"></waves-script></td>\n                        </tr>\n                    </tbody>\n                </table>\n                <div class="panel-body" ng-show="ctrl.scripts.length == 0">{{ctrl.scriptsMessage}}</div>\n            </uib-tab>\n        </uib-tabset>\n    </div>\n</div>\n');
```
[the remark this answers is not in the document]
Comment (username1565) [reply src/js/templates-1.1.8.js]: This files not autogenerated when I try to compile old wavesblockexplorer on windows 8.1.
Also, this not contains inside `index.html`.
So I just did copy this from https://wavesblockexplorer.com/ but now there is new wavesexplorer installed, after their server crush.
I did add this files to this repositary to do easy compile and run this all by double-click on `.bat`-files. See source code.

File: src/js/config-1.1.8.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+(function() {
```
[the remark this answers is not in the document]
Comment (username1565) [reply src/js/config-1.1.8.js]: This files not autogenerated when I try to compile old wavesblockexplorer on windows 8.1.
Also, this not contains inside `index.html`.
So I just did copy this from https://wavesblockexplorer.com/ but now there is new wavesexplorer installed, after their server crush.
I did add this files to this repositary to do easy compile and run this all by double-click on `.bat`-files. See source code.

**Comment by username1565** [comment]
You can try to build and run this old wavesblockexplorer from your repositary and from my.
I don't see any autogenerated files, when I try to build old explorer from the [`old-explorer` branch](https://github.com/wavesplatform/WavesExplorerLite/tree/old-explorer) in your [official repositary](https://github.com/wavesplatform/WavesExplorerLite/).

File: RUN_yarn-start.bat [hunk 5]
```diff
@@ -0,0 +1,4 @@
+@echo off
+cd %~dp0
+yarn start
```
Comment (username1565) [inline RUN_yarn-start.bat]: Just start using `yarn` the compiled wavesexplorer, after building it.

File: package.json [hunk 6]
```diff
@@ -21,7 +21,8 @@
     "ngclipboard": "^2.0.0",
     "restangular": "^1.6.1",
     "underscore": "^1.8.3",
-    "wavesplatform-core-js": "git://github.com/wavesplatform/wavesplatform.core.js.git#master"
+    "wavesplatform-core-js": "git://github.com/wavesplatform/wavesplatform.core.js.git#master",
+    "opn": "^6.0.0"
```
Comment (username1565) [inline package.json]: Add "opn" package, to make working the **open**ing wavesexplorer, by timeout, after start the server.
Just, like for new wavesexplorer.

File: server.js [hunk 7]
```diff
@@ -8,7 +8,7 @@ var morgan = require('morgan'); // formerly express.logger
 var errorhandler = require('errorhandler');
 var app = express();
 
-var basedir = __dirname + '/build';
+var basedir = __dirname + '/src';
```
Comment (username1565) [inline server.js]: Fix invalid pathway from "build" to "src"-folder.

File: server.js [hunk 8]
```diff
@@ -30,6 +30,16 @@ if ('development' === app.get('env')) {
   app.use(errorhandler());
 }
 
+var opn = require('opn');	//to open the url
```
Comment (username1565) [inline server.js]: This package was been added in package.json
require it, to open the wavesexplorer, after runnint the server.

File: server.js [hunk 9]
```diff
@@ -30,6 +30,16 @@ if ('development' === app.get('env')) {
...
+// opens the url in the default browser
 http.createServer(app).listen(app.get('port'), function () {
     console.log('Server listening on port ' + app.get('port'));
+	
+	setTimeout(
+		function(){
+			opn('http://'+'localhost:'+(process.env.PORT || 3000)+'/');			//open the url, after running server
+		},
+		1000
+	);
```
Comment (username1565) [inline server.js]: Open waves explorer on, after 1 second, after running the server. Open it on localhost, with default or specified port.

File: src/js/config-1.1.8.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (username1565) [reply src/js/config-1.1.8.js]: This file was been updated.

**Pull request closed without merging**